### DISCUSSION TITLE
Default communication Device

### DISCUSSION THREAD

**Discussion by MrkikouLIF** [File-New-Project/EarTrumpet#1149] [Ideas]
Hello,
When I switch my default audio device between my speakers and my headset. My communication software doesn't change because the default communication device is still on my headset. I have to change it manually each time I want to switch between my audio devices.
Is it possible to add an option to choose if we want to pilot both types of audio devices or not?
![image](https://user-images.githubusercontent.com/65592973/167441154-9eb420b8-a899-4b4d-8b0f-dbcfbfd945e3.png)

**Comment by pigsflew** [comment]
This is a super annoying thing for me, I almost always want my Default Audio Device and my Default Communication Audio Device (for both output *and* input) to be the same.

I'd love to be able to set the behavior to change both when I select one, esp. if they are the same currently.
